cspatients/serializers.py

```diff
@@ -11,7 +10,7 @@ class Meta:
 
 
 class PatientEntrySerializer(serializers.ModelSerializer):
-    decision_time = models.DateTimeField()
```

Comment:
Do we want to remove this field?

Reply:
Yes not sure why that was there in the first place.